Repository/RevisionRepository.php

```diff
@@ -609,6 +609,34 @@ public function lockRevisions(ContentType $contentType, \DateTime $until, $by, $
         return $qbUpdate->getQuery()->execute();
     }
 
+    public function unlockContentTypeRevisionsbyUser(ContentType $contentType, string $by): int
+    {
+        $params = ['content_type' => $contentType, 'by' => $by, 'null' => null];
+
+        /** @var QueryBuilder $qbSelect */
```

Comment:
please remove, $this->createQueryBuilder() has a correct doc block, so this is overkill :)